newdle/client/src/client.js

```diff
@@ -181,7 +179,7 @@ class Client {
     try {
       resp = await fetch(url, requestOptions);
     } catch (err) {
-      throw new ClientError(url, 0, err);
+      this.store.dispatch(addError(err.message));
```

Comment:
Now you aren't aborting execution of the code below anymore...

Reply:
also, during development we now have much less information on what went wrong. I think we should at least `console.error()` the actual ClientError...